data/assets/scene/digitaluniverse/stars.asset

```diff
@@ -32,13 +32,19 @@ local Stars = {
   Renderable = {
     Type = "RenderableStars",
     File = speck .. "stars.speck",
-    Texture = textures .. "halo.png",
-    --ShapeTexture = textures .. "disc.png",
+    Halo = {
+      Texture = textures .. "halo.png",
+      PartialOpacity = 0.875
+    },
+    Glare = {
+      Texture = textures .. "glare.png",
+      PartialOpacity = 5.0,
```

Comment:
Why does the partial opacity go to more than 1?  Confused :)

Reply:
Agreed.  Renamed it to "Multiplier"